how-to-guides/KyloDatasources.rst

```diff
@@ -0,0 +1,80 @@
+========================
+Kylo Datasources Guide
+========================
+
+Introduction
+====================
+
+Kylo can manage the creation and usage of Nifi RDBMS data source configurations, through a simple `Data Source UI <http://localhost:8400/index.html#!/datasources>`_.
+
+JDBC
+=============
+
+.. note:: Permissions for the jars are separate per the type of process running.
+
+Locations
+-----------
+
+    +----------------------------------+------------------------------------------------------------------------------------------------------------------+
+    | $NIFI_HOME/nifi-<version>/lib    | Needed by Nifi for the DBCPConnectionPool. Can set a custom path, but need to set `Database Driver Location(s)`  |
+    +----------------------------------+------------------------------------------------------------------------------------------------------------------+
+    | $KYLO_HOME/kylo-services/plugin  | Needed by Kylo in the schema discovery (Data Ingestion)                                                          |
+    +----------------------------------+------------------------------------------------------------------------------------------------------------------+
+    | $KYLO_HOME/kylo-services/lib     | Needed by Kylo wrangler (Visual Query / Data Transformation) with Spark SQL                                      |
+    +----------------------------------+------------------------------------------------------------------------------------------------------------------+
+    | $SPARK_HOME/lib/                 | Needed by Spark to be distributed on the cluster, for the data wrangling                                         |
```

Comment:
Do we have a suggestion to where can simply dump any additional jars needed by spark, which might be picked up automatically?

I would like to minimize any template modification for each possible client.